packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2934,6 +2934,7 @@ class TextSelectionGestureDetectorBuilder {
     _dragStartViewportOffset = 0.0;
     _dragStartScrollOffset = 0.0;
     if (defaultTargetPlatform == TargetPlatform.iOS &&
+        _isEditableTextMounted &&
```

Comment:
super nit: let `_isEditableTextMounted` be at the front of the logic in this if branch.